webapp/src/main/webapp/src/assets/i18n/en.json

```diff
@@ -70,6 +70,18 @@
           "id": "Id",
           "file-name": "File Name",
           "created": "Created"
+        },
+        "warning-html": "<p><strong class=\"mr-xs\">WARNING:</strong> Groups in the uploaded file will replace existing groups if they have the same school ID, school year and group name.</p><p>Existing groups will not be deleted and will remain unchanged.</p>",
+        "file-format": {
+          "header": "CSV File Format",
+          "template": {
+            "button": "Template",
+            "file": "groups.csv"
+          },
+          "rules": {
+            "header": "How to write the CSV",
+            "body-html": "<ol> <li class=\"mt-md\"> The CSV file must include the headers found in the template file provided below. </li><li class=\"mt-md\"> Every row must include a valid <code>school_natural_id</code>, <code>school_year</code>, and <code>group_name</code>. </li><li class=\"mt-md\"> Schools must be grouped together and not separated by rows of groups for other schools. </li><li class=\"mt-md\"> Groups must be grouped together and not separated by rows of other groups. </li><li class=\"mt-md\"> A group's subject must be consistent, and either empty, <code>Math</code>, <code>ELA</code> or <code>All</code>. </li><li class=\"mt-md\"> An empty group subject will be interpreted as <code>All</code>. </li><li class=\"mt-md\"> You cannot assign more than <code>200</code> students to a single group. </li><li class=\"mt-md\"> You must have <code>GROUP_WRITE</code> access to a group's school in order to upload the group. </li></ol>"
+          }
```

Comment:
For these; 'grouped together' is that clear, especially 'Groups must be grouped'
  Schools must be grouped together and not separated by rows of groups for other schools. 
  Groups must be grouped together and not separated by rows of other groups.